standard_hostpool.go

```diff
@@ -0,0 +1,290 @@
+package hostpool
+
+import (
+	"log"
+	"sync"
+	"time"
+)
+
+// standardHostPoolResponse implements HostPoolResponse
+type standardHostPoolResponse struct {
```

Comment:
(Minor nitpick, feel free to ignore) but maybe we could reorder these types? It surprises me to see the response at the top as I'm not immediately sure what that's for.